activemq-broker/src/main/java/org/apache/activemq/broker/jmx/HealthView.java

```diff
@@ -167,6 +169,25 @@ public List<HealthStatus> healthList() throws Exception {
             }
         }
 
+        /**
+         * Check The Transport Connector limits
+         */
+
+        if (brokerService != null && !brokerService.getTransportConnectors().isEmpty()) {
+        	for(TransportConnector tc: brokerService.getTransportConnectors()) {
+        		if(tc.getServer() instanceof TcpTransportServer) {
+        			int connectionUsage = (int) (((TcpTransportServer)tc.getServer()).getCurrentTransportCount().get() * 100 ) / ((TcpTransportServer)tc.getServer()).getMaximumConnections();
+        			if(((TcpTransportServer)tc.getServer()).getCurrentTransportCount().get() >= ((TcpTransportServer)tc.getServer()).getMaximumConnections()) {
+        				String message = "Exceeded the maximum number of allowed client connections: " + ((TcpTransportServer)tc.getServer()).getMaximumConnections();
```

Comment:
This check can’t happen since the connector will close a connection when it reaches the limit. An exceeded count isn’t possible. 

I think this branch of logic should just be removed and simply go with the limit check

Reply:
Agree, just the limit is enough.

We can relay on the existing metrics for that.